modules/gdscript/gdscript_analyzer.cpp

```diff
@@ -4838,18 +4838,33 @@ void GDScriptAnalyzer::validate_call_arg(const List<GDScriptParser::DataType> &p
 
 		if ((arg_type.is_variant() || !arg_type.is_hard_type()) && !(par_type.is_hard_type() && par_type.is_variant())) {
 			// Argument can be anything, so this is unsafe.
+#ifdef DEBUG_ENABLED
 			mark_node_unsafe(p_call->arguments[i]);
```

Comment:
So all of the `push_warning()` that I saw in the analyzer were protected by `DEBUG_ENABLED`, but none of the `mark_node_unsafe()` calls were. I think it might make sense to pull this one outside the `#ifdef` block here?

Reply:
https://github.com/godotengine/godot/blob/f6bf51ca49ed0073a3c34054a7ab583faff2d3ec/modules/gdscript/gdscript_analyzer.cpp#L5117-L5127